app/panel/components/Stats.jsx

```diff
@@ -329,7 +329,7 @@ class Stats extends React.Component {
 			monthlyAverageData: clearData,
 			dailyAverageData: clearData,
 			showResetModal: false,
-			showPitchModal: (!this.props.user || !this.props.user.subscriptionsPlus),
+			showPitchModal: (!this.props.user || !this.props.user.plusAccess),
```

Comment:
While you're in there, can you destructure `this.props.user` like you've done elsewhere?

Reply:
Done!